client/my-sites/checkout/checkout-thank-you/style.scss

```diff
@@ -680,6 +680,8 @@
 		display: flex;
 		flex-direction: column;
 		padding: 0;
+		border-radius: calc( 2px * 2 );
```

Comment:
Just out of curiosity, what difference makes using `calc( 2px * 2)` instead of `4px`?

Reply:
A tip I got from @elliottprogrammer — I assume it's because the linter doesn't like `4px` and this is a nice way to go around that. This brings up two good questions thought:

- Does the `calc( 2px * 2 )` make it less instantly readable and is that a concern?
- If we're going around the linter, should we instead add the comment to make that explicit?

Reply:
> Does the calc( 2px * 2 ) make it less instantly readable and is that a concern?
I don't think so. It just makes you wonder why 🙂.

> If we're going around the linter, should we instead add the comment to make that explicit?
I think that would be better because it is more explicit.

Reply:
cc @elliottprogrammer FYI, I'm adding the `/* stylelint-disable-line */` to this one.

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/54599/commits/472e2c39a7b9a24a2b854a4a901d1a05ebfc9446